x/operations/ante/ante.go

```diff
@@ -20,7 +20,9 @@ func (mgp MinGasPriceDecorator) AnteHandle(
 	simulate bool,
 	next sdk.AnteHandler,
 ) (newCtx sdk.Context, err error) {
-	price := mgp.keeper.GetParams(ctx.WithGasMeter(sdk.NewInfiniteGasMeter())).MinGasPrice
-	ctx = ctx.WithMinGasPrices(sdk.NewDecCoins(price))
+	price := mgp.keeper.GetMinGasPrice(ctx.WithGasMeter(sdk.NewInfiniteGasMeter()))
```

Comment:
in the updated sdk the InitGenesis func calls ante. It caused a panic since params store was empty (genesis not fully loaded)